Revision: 7a698e11296b7e599206dc39af89717ec134f8ea
Patch-set: 1
File: /COMMIT_MSG

18:6-18:8
Thu Jun 23 11:12:04 2016 +0000
Author: Nicolas Geoffray <1038443@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: ce721291_d454fc3e
Bytes: 2
it

18:6-18:8
Thu Jun 23 12:59:34 2016 +0000
Author: Vladimir Marko <1018108@85c56323-6fa9-3386-8a01-6480fb634889>
Parent: ce721291_d454fc3e
UUID: 8e657ac6_da1c256e
Bytes: 4
Done

File: test/004-ThreadStress/check

17:24-17:25
Thu Jun 23 11:12:04 2016 +0000
Author: Nicolas Geoffray <1038443@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: 8e657ac6_5a8655c4
Bytes: 30
, so replace numbers with 'N'.

17:24-17:25
Thu Jun 23 12:59:34 2016 +0000
Author: Vladimir Marko <1018108@85c56323-6fa9-3386-8a01-6480fb634889>
Parent: 8e657ac6_5a8655c4
UUID: 2e572eeb_cf99bde5
Bytes: 4
Done

File: test/004-ThreadStress/src/Main.java

405:0-405:68
Thu Jun 23 11:12:04 2016 +0000
Author: Nicolas Geoffray <1038443@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: 2e300ecd_47704ef1
Bytes: 212
Won't that hide the failure? I guess the expected.txt test will fail, but would it make sense to also do System.exit(errorCode) and then make sure our run-test understand that such error code makes the test fail?

405:0-405:68
Thu Jun 23 12:59:34 2016 +0000
Author: Vladimir Marko <1018108@85c56323-6fa9-3386-8a01-6480fb634889>
Parent: 2e300ecd_47704ef1
UUID: ce205285_8708887d
Bytes: 185
Do we check the exit code?

If there are still threads trying to perform tasks (including OOM, Alloc and LargeAlloc), there is no guarantee we shall ever be able to print anything here.

499:0-499:4
Thu Jun 23 11:12:04 2016 +0000
Author: Nicolas Geoffray <1038443@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: 0e3eea9b_4e7073f1
Bytes: 8
extra ws

499:0-499:4
Thu Jun 23 12:59:34 2016 +0000
Author: Vladimir Marko <1018108@85c56323-6fa9-3386-8a01-6480fb634889>
Parent: 0e3eea9b_4e7073f1
UUID: 2e572eeb_cf161d3d
Bytes: 23
Already removed in PS2.

508:57-509:71
Thu Jun 23 11:12:04 2016 +0000
Author: Nicolas Geoffray <1038443@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: 0e3eea9b_ce54633f
Bytes: 54
Add a comment why it's ok failing creating the Thread.

508:57-509:71
Thu Jun 23 12:59:34 2016 +0000
Author: Vladimir Marko <1018108@85c56323-6fa9-3386-8a01-6480fb634889>
Parent: 0e3eea9b_ce54633f
UUID: 2e572eeb_af9479ae
Bytes: 65
Added some comment that indirectly implies that it's "almost OK".

653:0-654:39
Thu Jun 23 11:12:04 2016 +0000
Author: Nicolas Geoffray <1038443@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: 8e657ac6_fa4f2977
Bytes: 90
Please add a comment on why you're doing this. I guess that's to make sure the test fails?

653:0-654:39
Thu Jun 23 12:59:34 2016 +0000
Author: Vladimir Marko <1018108@85c56323-6fa9-3386-8a01-6480fb634889>
Parent: 8e657ac6_fa4f2977
UUID: 0e798a73_f654a0e5
Bytes: 82
Yes, rather than allowing the main thread to be blocked indefinitely on a Barrier.

668:63-668:68
Thu Jun 23 11:12:04 2016 +0000
Author: Nicolas Geoffray <1038443@85c56323-6fa9-3386-8a01-6480fb634889>
UUID: 0e3eea9b_ee591f74
Bytes: 5
will?

668:63-668:68
Thu Jun 23 12:59:34 2016 +0000
Author: Vladimir Marko <1018108@85c56323-6fa9-3386-8a01-6480fb634889>
Parent: 0e3eea9b_ee591f74
UUID: 430b5d18_7856611e
Bytes: 89
This spells out a requirement on the runtime. As such I prefer the authoritative "shall".

